client/src/pages/LessonHome/LessonHome.js

```diff
@@ -4,9 +4,12 @@ import PropTypes from 'prop-types'
 
 import { useSelector, useDispatch } from 'react-redux'
 import { setField, resetField } from 'slices/language.slice'
-import { getLesson, downloadImageFile } from 'api'
+import { getLesson, downloadImageFile, downloadAudioFile } from 'api'
 import { useErrorWrap, useTrackPromise } from 'hooks'
 
+// eslint-disable-next-line no-unused-vars
+import _, { clone } from 'lodash'
```

Comment:
Maybe I'm missing something here, but if you can you should try not to have unused variables for this.